src/client/clientlauncher.cpp

```diff
@@ -53,17 +53,13 @@ MainGameCallback *g_gamecallback = NULL;
 
 ClientLauncher::~ClientLauncher()
 {
-	if (receiver)
-		delete receiver;
+	delete receiver;
 
-	if (input)
-		delete input;
+	delete input;
 
-	if (g_fontengine)
-		delete g_fontengine;
+	delete g_fontengine;
 
-	if (device)
```

Comment:
* I find it not so interesting at the moment what is performed by the called member function.
  Is it more important to determine if this function may be called over a pointer if it would not be checked before?
* How much do distinguish sanity checks?